app/views/devise/passwords/new.html.erb

```diff
@@ -0,0 +1,16 @@
+<h2>Forgot your password?</h2>
+
+<%= form_for(resource, as: resource_name, url: password_path(resource_name), html: { method: :post }) do |f| %>
+  <%= devise_error_messages! %>
+
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true %>
+  </div>
+
+  <div class="actions">
+    <%= f.submit "Send me reset password instructions" %>
+  </div>
```

Comment:
CSS is hijacking your submit button and causing to render as a second magnifying glass by the search box.

![screen shot 2016-01-22 at 11 16 34 am](https://cloud.githubusercontent.com/assets/3615206/12516161/06fc40f6-c0fa-11e5-9931-fea9ce6d300f.png)